### ISSUE TITLE
Make it clearer that people can join late if not everyone has joined on time

### ISSUE THREAD

**Issue by williamcodes** [stride-nyc/remote_retro#591]
As a facilitator,
I want to know that people can join late,
So that I know to start the meeting if it's been a while even when some people haven't joined yet.

__By Way Of Explanation:__
I keep noticing new facilitators stalling for time in large meetings when it takes forever for everyone to join. One facilitator complained today, "At least in Trello you can start the meeting on time and people can join late"

__Acceptance Criteria:__

- [ ] There is some explainer text for the facilitator so they know this important detail

**Comment by vanderhoop** [comment]
yeah, this is a great callout, @williamcodes. gonna see if the bench folks can arrive at better language
